src/webgpu/api/operation/render_pipeline/sample_mask.spec.ts

```diff
@@ -339,39 +356,168 @@ class F extends GPUTest {
       B: 0,
       A: 0,
     };
+    if (sampleCount === 1) {
+      if (
+        // rasterization needs to cover the center of the pixel
+        (rasterizationMask & 0b1111) >= 0b1111 &&
```

Comment:
Agree with statement. Now logic of sampleCount === 1 and sampleCount > 1 should have merged mostly. The only branching now is when sampleCount === 1, the center is sampling from Texel[3] yellow, instead of Texel[0] red (sampleIndex === 0)